src/plugins/mf_priority.cpp

```diff
@@ -918,7 +918,8 @@ static int validate_cb (flux_plugin_t *p,
                                            args,
                                            "cannot find user/bank or "
                                            "user/default bank entry "
-                                           "for uid: %i", userid);
+                                           "for uid: %i",
+                                           userid);
```

Comment:
Is there an automatic formatter for C and C++ code for this repo?

Reply:
There is not, but @wihobbs has made that suggestion before too. I should look into adding one! I can open an issue on this to track it.